src/App.css

```diff
@@ -1,6 +1,295 @@
-/*
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;900&display=swap');
 
-body{
-  margin:30px;
+html,
+body,
+div,
+span,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ul,
+li,
+form,
+label,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+menu,
+nav,
+section {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+}
 
-  */
+* {
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.navbar {
+  display: flex;
+  align-items: flex-end;
+  padding: 1rem;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  width: 100%;
+}
+
+nav {
+  margin-top: -43px;
+}
+
+.logo {
+  font-weight: bold;
+  font-size: 1.875rem;
+}
+
+.navbar li {
+  list-style: none;
+  margin: 0.3rem 2.5rem;
+  color: #0290ff;
+}
```

Comment:
Please note that the project requirements state that all elements in your design should match the zeplin design. Please update your app to look more like the zeplin design:

Your design:
<img width="497" alt="image" src="https://github.com/harshi0102/book-store/assets/37116322/e4e9da35-9884-4bb4-bb5e-01bfb397d6e8">

Zeplin design:
<img width="498" alt="image" src="https://github.com/harshi0102/book-store/assets/37116322/0a3aa19f-96aa-49ea-9e87-26194f31300f">


A few things to note:
- Please add the padding around the main container and make sure it has the same background color
- Kindly ensure the navbar links are centre-aligned and use the same font weight and color
- Kindly make sure the text elements in the card use the same font weight and color
- Kindly reduce the spacing between each card
- Kindly update the background color of the section with the cards to match the zeplin
- Kindly add the divider between the card section and the form and space out the form title from the form inputs